lib/record.rb

```diff
@@ -0,0 +1,23 @@
+class Record
+  attr_reader :amount, :date, :type
+
+  def initialize(amount, date)
+    @amount = amount
+    @date = date
+    @type = set_type
+  end
+
+  def add_to(ledger)
```

Comment:
This feels a little inside out.  Should a record add itself to a ledger, or should the ledger add a transaction to itself?